kedro-datasets/kedro_datasets/ibis/table_dataset.py

```diff
@@ -186,7 +186,7 @@ def _describe(self) -> dict[str, Any]:
             "filepath": self._filepath,
             "file_format": self._file_format,
             "table_name": self._table_name,
-            "connection_config": self._connection_config,
+            "backend": self._connection_config.get("backend") if self._connection_config else None,
```

Comment:
Makes sense, since it avoids constructing the connection! Initially, I was wondering why not just use `self.connection`, but this is better. :)